src/v2/guide/single-file-components.md

```diff
@@ -1,48 +1,48 @@
 
-Todos estos puntos estan resueltos por **single-file components** con extensión `.vue`, lo cual es posible con herramientas como Webpack or Browserify.
+Todo esto se resuelve mediante Componentes de un Solo Archivo con extensión .vue, lo cual es posible con herramientas de compilación como Webpack o Browserify.
 
 Aquí tenemos un ejemplo de un archivo que llamaremos `Hello.vue`:
 
-<a href="https://gist.github.com/chrisvfritz/e2b6a6110e0829d78fa4aedf7cf6b235" target="_blank"><img src="/images/vue-component.png" alt="Single-file component example (click for code as text)" style="display: block; margin: 30px auto;"></a>
+<a href="https://gist.github.com/chrisvfritz/e2b6a6110e0829d78fa4aedf7cf6b235" target="_blank"><img src="/images/vue-component.png" alt="Ejemplo de Componente de un Solo Archivo (click para ver código como texto)" style="display: block; margin: 30px auto;"></a>
 
 Ahora tenemos:
 
 - [Resaltado completo de la sintaxis](https://github.com/vuejs/awesome-vue#source-code-editing)
-- [Módulos CommonJSCommonJS modules](https://webpack.js.org/concepts/modules/#what-is-a-webpack-module)
-- [Definición de alcance del CSS por Component](https://vue-loader.vuejs.org/en/features/scoped-css.html)
+- [Módulos CommonJS](https://webpack.js.org/concepts/modules/#what-is-a-webpack-module)
+- [CSS con alcance de componente](https://vue-loader.vuejs.org/en/features/scoped-css.html)
 
-Como prometimos, podemos también usar preprocesadores como Pug, Babel (con módulos ES2015), y Stylus para componentes más limpios y con más características.
+Como prometimos, podemos también usar pre-procesadores como Pug, Babel (con módulos ES2015) y Stylus para obtener componentes más limpios y con más funcionalidades.
 
-<a href="https://gist.github.com/chrisvfritz/1c9f2daea9bc078dcb47e9a82e5f7587" target="_blank"><img src="/images/vue-component-with-preprocessors.png" alt="Single-file component example with preprocessors (click for code as text)" style="display: block; margin: 30px auto;"></a>
+<a href="https://gist.github.com/chrisvfritz/1c9f2daea9bc078dcb47e9a82e5f7587" target="_blank"><img src="/images/vue-component-with-preprocessors.png" alt="Ejemplo de Componente de un Solo Archivo con pre-procesadores (click para ver código como texto)" style="display: block; margin: 30px auto;"></a>
 
-Estos lenguajes específicos son solo ejemplos. Usted podrías usar facilmente Bublé, TypeScript, SCSS, PostCSS - o cualquier otro preprocesador que le ayude a ser productivo. Si utiliza Webpack con `vue-loader`, esto también tiene soporte de primera clase para módulos CSS.
+Estos lenguajes específicos son solo ejemplos. Podrías usar, con la misma facilidad, Bublé, TypeScript, SCSS, PostCSS, o cualquier otro pre-procesador que te ayude a ser productivo. Si utilizas Webpack con `vue-loader`, también te ofrece soporte de primera clase para módulos CSS.
```

Comment:
We should use the third form here:

```suggestion
Estos lenguajes específicos son solo ejemplos. Usted podría usar, con la misma facilidad, Bublé, TypeScript, SCSS, PostCSS, o cualquier otro pre-procesador que le ayude a ser productivo. Si usted utiliza Webpack con `vue-loader`, también le ofrece soporte de primera clase para módulos CSS.
```

Reply:
Changed to formal treatment throughout the document.